pkg/reconciler/apis/apiexport/apiexport_controller.go

```diff
@@ -81,13 +82,13 @@ func NewController(
 			return namespaceInformer.Lister().Get(clusters.ToClusterAwareKey(clusterName, name))
 		},
 		createNamespace: func(ctx context.Context, clusterName logicalcluster.Name, ns *corev1.Namespace) error {
-			_, err := kubeClusterClient.Cluster(clusterName).CoreV1().Namespaces().Create(ctx, ns, metav1.CreateOptions{})
+			_, err := kubeClusterClient.CoreV1().Namespaces().Create(kcpclienthelper.WithCluster(ctx, clusterName), ns, metav1.CreateOptions{})
```

Comment:
that's a long alis, making these lines much longer. Anything short we can come up with as a convention?

Reply:
How about `kcphelper`? I'm open to other alternatives too. Another apt one would be `kcpapimachinery` but that would made the lines really long.

Reply:
Addressed in https://github.com/kcp-dev/kcp/pull/1376#discussion_r911498030

Reply:
@varshaprasad96 it looks like the comment you linked is about the rest.Config, not the context?

Reply:
What about logicalcluster.Wrap(ctx, clusterName) / logicalcluster.FromContext(ctx)?